### DISCUSSION TITLE
We're participating in Hacktoberfest'23!

### DISCUSSION THREAD

**Discussion by kehoecj** [Boeing/config-file-validator#49] [Announcements]
Thank you to the OSS Community for the 80+ GitHub Stars and all the [contributors](https://github.com/Boeing/config-file-validator/graphs/contributors) who have added features to the config-file-validator tool. You have made such a big impact on this project and we really appreciate it!

We wanted to announce that we're participating in Hacktoberfest 2023 and highlight some ways that you can participate.

### Work an existing issue
We have gone through and marked all of our issues that we felt were good candidates for hacktoberfest with the https://github.com/Boeing/config-file-validator/labels/hacktoberfest label. These issues should be fairly straightforward to implement without much or any code refactoring. If there is an issue that you'd like to work that is not labeled https://github.com/Boeing/config-file-validator/labels/hacktoberfest you can still work that too. To work an issue follow the steps below:

1. Comment on the issue that you'd like to work it. We'll assign the issue first-come first serve. If you later decide you can't work it please let us know so that we can reassign the issue.
2. During development use the issue to ask any clarifying questions, start architecture discussions, etc
3. After development is complete submit the PR for review. Your PR will be reviewed by two maintainers and you will need to resolve all the comments.
4. Once the review is complete the maintainers will merge it and add the https://github.com/Boeing/config-file-validator/labels/hacktoberfest-accepted label

### Create and work an issue
If you have an idea that is not on the existing issue list that you'd like to work please submit it! The process will be similar to working an existing issue with a few additional steps.

1. Create a [new issue](https://github.com/Boeing/config-file-validator/issues/new) with your idea or bug and mark it with the https://github.com/Boeing/config-file-validator/labels/hacktoberfest label. Be as descriptive as possible so that the maintainers can understand the issue.
2. The maintainers will review the issue and ask any clarifying questions
3. Once all questions have been resolved the maintainers will assign you the issue
4. During development use the issue to ask any clarifying questions, start architecture discussions, etc
5. After development is complete submit the PR for review. Your PR will be reviewed by two maintainers and you will need to resolve all the comments.
6. Once the review is complete the maintainers will merge it and add the https://github.com/Boeing/config-file-validator/labels/hacktoberfest-accepted label

The maintainers will stay on top of the issues and PR's so that the OSS community can get timely feedback and credit for their hacktoberfest submissions.

Thank you in advance and let us know if there are any questions!

**Comment by kehoecj** [comment]
First https://github.com/Boeing/config-file-validator/labels/hacktoberfest PR has been approved and merged. Thank you @sandyydk for your contribution!

**Comment by sandyydk** [comment]
Thanks @clayton for the support. Glad to contribute.

**Comment by gokulav137** [comment]
I was able to complete hacktoberfest with the four contributions I made to config-file-validator 🎉.

Big thanks to the maintainers for their support @kehoecj @jd4235 @jackswiney ❤️.

**Reply by kehoecj** [reply]
@gokulav137 Thank _you_ for your contributions! We're so glad you picked the config-file-validator for https://github.com/Boeing/config-file-validator/labels/hacktoberfest
